src/databricks/labs/remorph/reconcile/connectors/data_source.py

```diff
@@ -20,11 +20,16 @@ def __init__(self, source: str, spark: SparkSession, ws: WorkspaceClient, scope:
         self.scope = scope
 
     @abstractmethod
-    def read_data(self, schema_name: str, catalog_name: str, query: str, table_conf: Tables) -> DataFrame:
+    def read_data(self, catalog: str, schema: str, query: str, jdbc_reader_options: JdbcReaderOptions) -> DataFrame:
```

Comment:
```suggestion
    def read_data(self, catalog: str, schema: str, query: str, options: JdbcReaderOptions) -> DataFrame:
```
nit: can you still rename all arguments to make them reasonably shorter? :)

Reply:
updated the arguments to shorter.